File: lib/models/movie_track.dart
class MovieTrack {
  final String episodeKey;
  final String element;
  final bool isMegogo;
  final String movieKey;
  final String seasonKey;
  final int seconds;
  final String userId;

  MovieTrack({
    required this.episodeKey,
    required this.element,
    required this.seasonKey,
    required this.isMegogo,
    required this.movieKey,
    required this.seconds,
    required this.userId,
  });

  Map<String, dynamic> toJson() {
    final map = <String, dynamic>{};
    map['episode_key'] = episodeKey;
    map['is_megogo'] = isMegogo;
    map['movie_key'] = movieKey;
    map['season_key'] = seasonKey;
    map['seconds'] = seconds;
    map['user_id'] = userId;
    map['element'] = element;
    return map;
  }
}

File: android/src/main/kotlin/uz/udevs/udevs_video_player/models/MovieTrack.kt
package uz.udevs.udevs_video_player.models

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class MovieTrack(
    @SerializedName("episode_key")
    val episodeKey: String,
    @SerializedName("is_megogo")
    val isMegogo: Boolean,
    @SerializedName("movie_key")
    val movieKey: String,
    @SerializedName("season_key")
    val seasonKey: String,
    @SerializedName("seconds")
    val seconds: Int,
    @SerializedName("user_id")
    val userId: String,
    @SerializedName("element")
    val element: String,
) : Serializable